packages/as-transformer/src/transformers/massaExport.ts

```diff
@@ -85,6 +110,18 @@ export function generateWrapper(
   return wrapper;
 }
 
+/**
+ * Generates the imports for the wrapper functions.
+ *
+ * @remarks
+ * The imports are the AS helpers generated by the protobuf files.
```

Comment:
As I understand generateASHelpers, the helpers are generated by protoc from the proto files